vtrace/tests/tests_envitools.py

```diff
@@ -1,13 +1,26 @@
 import vtrace
 import vtrace.envitools
+import vtrace.util
 
 
 class EnvitoolsTests(vtrace.tests.VtraceProcessTest):
     def test_emulatorFromTrace(self):
-        emu = vtrace.envitools.emuFromTrace(self.trace)
+        emu = vtrace.util.emuFromTrace(self.trace)
 
         ctx = self.trace.getRegisterContext()
         for rname, idx in ctx.getRegisterNameIndexes():
             self.assertEqual(self.trace.getRegister(idx), emu.getRegister(idx))
 
         emu.stepi()
+
+    def test_workspaceFromTrace(self):
```

Comment:
Can you also test the strict option since I'm still not sure where/how it's used?